### ISSUE TITLE
Autosync

### ISSUE THREAD

**Issue by DGKSK8LIFE** [DGKSK8LIFE/redisql#23]
Could be implemented in a redis-native way with Redis pub/sub; not sure about MySQL

**Comment by ShlokMohanty** [comment]
@DGKSK8LIFE may i kow what are we wanting to do  an Async of ?

**Comment by DGKSK8LIFE** [comment]
I basically want to find a way to do auto sync (meaning that, when a change happens in a SQL database, it is reflected in Redis automatically, sort of like an "event-driven" architecture), integrating into what I've already written (the internal `Convert` function here: https://github.com/DGKSK8LIFE/redisql/blob/master/utils/db.go).

**Comment by ShlokMohanty** [comment]
@DGKSK8LIFE ok got it can i take on this issue if y9ou could plz assign this task to me ?

**Comment by DGKSK8LIFE** [comment]
Sure, will assign others if they'd like to though.

**Comment by ShlokMohanty** [comment]
thank you @DGKSK8LIFE  for assigning this issue to me .

**Comment by DGKSK8LIFE** [comment]
Any updates @ShlokMohanty? This doesn't necessarily have to be implemented in code, more of an architectural question (same for #28)

**Comment by ShlokMohanty** [comment]
yeah @DGKSK8LIFE i was looking at the redis document would be implementing it from tomorrow or day after tomorrow will it be fine for you ?

**Comment by DGKSK8LIFE** [comment]
No pressure @ShlokMohanty! Take as long as you need, just wondering if you had any ideas for a potential implementation; cause my ideas would perform badly if implemented.

**Comment by ShlokMohanty** [comment]
@DGKSK8LIFE could we use Mysql for the autosync ?

**Comment by ShlokMohanty** [comment]
if i get some resources like how to do it then I will surely inform you

**Comment by DGKSK8LIFE** [comment]
@ShlokMohanty It should support MySQL and Postgres; test with whatever u prefer.

**Comment by DGKSK8LIFE** [comment]
The internal Convert() function already supports Postgres and MySQL out of the box so autosync should be built on top of it

**Comment by ShlokMohanty** [comment]
ok sure then it would be great

**Comment by DGKSK8LIFE** [comment]
@ShlokMohanty No problem. Take all the time you need. Perhaps join the discord server so that we can communicate more about the autosync idea and implementation. Sorry, I haven't done the best job of explaining the codebase as it is currently.

**Comment by DGKSK8LIFE** [comment]
Yes @ShlokMohanty. It is also in the READMEz https://discord.gg/RfZU9dp9ts
